migrations/V24__Add_Actions_Table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TYPE action_type AS ENUM ('SUBMIT');
+
+CREATE TABLE actions (
+    id uuid primary key not null,
+    action_type action_type not null,
+    actor_name text not null CHECK (actor_name <> ''),
+    actor_email text not null CHECK (actor_name <> ''),
+    actor_eua_user_id text not null CHECK (actor_eua_user_id ~ '^[A-Z0-9]{4}$'),
+    created_at timestamp with time zone,
+    intake_id uuid references system_intake(id),
+    business_case_id uuid references business_case(id) check (num_nonnulls(intake_id, business_case_id) = 1)
```

Comment:
Open question: should system intake actions and business case actions be their own tables? They use most of (maybe all of) the same fields. I don't think it's a good idea to combine these references into a single field lest we lose the database structure of the formal reference.

Reply:
Also: We won't be updating actions, right? So do we need an updated_at column?